pkgs/tools/admin/oci-cli/default.nix

```diff
@@ -73,6 +76,25 @@ buildPythonApplication rec {
       --replace "oci==2.78.0" "oci"
   '';
 
+  postInstall = ''
+    # Similar to the gcloud CLI
```

Comment:
There is no package named gcloud, also such comments are not that useful in the future

```suggestion
```